cmd/kubeadm/app/cmd/reset.go

```diff
@@ -220,21 +210,6 @@ func newCmdReset(in io.Reader, out io.Writer, resetOptions *resetOptions) *cobra
 	return cmd
 }
 
-func cleanDirs(data *resetData) {
-	if data.DryRun() {
-		fmt.Printf("[reset] Would delete contents of stateful directories: %v\n", data.dirsToClean)
-		return
-	}
```

Comment:
if i'm not mistaken CleanDir has no dryrun mode and we are losing the dry run ability here.

the verbose output `[reset] Would ...` is useful too.

Reply:
> if i'm not mistaken CleanDir has no dryrun mode and we are losing the dry run ability here. 

Good point!

For the `cleanup-node`, it is already guarded [here](https://github.com/kubernetes/kubernetes/pull/110972/commits/de1223419a600c36fdc424bc86cd5d07012ea8f9#diff-6b1b1f72cbc95d2c92bb0f2f82dda888b5c46459ea5cb2c33dba692d6e182d88R149)

For the `remove-etcd-member`, We can add a check before the `CleanDir`, it's [here](https://github.com/kubernetes/kubernetes/pull/110972/files#diff-2a862772a2b5bf076910e45b95c7c1da6a457581228efb1f10f763fc1e17d9bfR74)

> the verbose output [reset] Would ... is useful too.

We have the printed the message [here](https://github.com/kubernetes/kubernetes/pull/110972/files#diff-6b1b1f72cbc95d2c92bb0f2f82dda888b5c46459ea5cb2c33dba692d6e182d88R151), but now it doesn't tell whether it is stateful dir or now.